Packs/Gmail/Playbooks/playbook-Search_And_Delete_Emails_-_Gmail.yml

```diff
@@ -1,191 +1,164 @@
-description: This playbook searches Gmail to identify and delete emails with similar
```

Comment:
Weren't we suppose to provide a disclaimer here about the google search limitations?

Reply:
Added one.
@julieschwartz18 Can you please review: https://github.com/demisto/content/pull/16327/commits/b3a85589394f9d30c9f638431d025d07d9ffb03b